tools/runme/build.sh

```diff
@@ -109,13 +109,17 @@ _sbt_build() {
     TESTS=",$TESTS,"; TESTS="${TESTS//,+scala,/,}"; TESTS="${TESTS#,}"; TESTS="${TESTS%,}"
     if [[ "$TESTS" = "" ]]; then TESTS="all"; fi
   fi
-  _sbt_run "full-build"
-  show section "Sorting assembly jar for the maven repo"
-  # leave only the -assembley jars under the proper name (and the pom files)
-  local f; for f in "$BUILD_ARTIFACTS/packages/m2/"**; do case "$f" in
-    ( *-@(javadoc|sources).jar@(|.md5|.sha1) ) _rm "$f" ;;
-    ( *-assembly.jar@(|.md5|.sha1) ) _ mv "$f" "${f//-assembly.jar/.jar}" ;;
-  esac; done
+  if [[ "$SUBPROJECT" != "" ]]; then 
+      _sbt_run "project $SUBPROJECT" compile
```

Comment:
Can we revert this change? Ill take care of cleaning up the build in a separate large PR

Reply:
when i ran `sbt 'project lightbm' compile`, it appeared than many variables was missing. 
After reviewing the build.sh I saw it will make some exporting of the variables to the env,
so I changed the build.sh.
I will revert the changes and you do the fixing, thank you !